runtime/expr/function/compare.go

```diff
@@ -0,0 +1,28 @@
+package function
+
+import (
+	"github.com/brimdata/zed"
+	"github.com/brimdata/zed/runtime/expr"
+)
+
+func NewCompare() *Compare {
+	return &Compare{
+		fn: expr.NewValueCompareFn(true),
+	}
+}
+
+type Compare struct {
```

Comment:
1. Include a doc comment.
2. Move this before the constructor.
```suggestion
// https://github.com/brimdata/zed/blob/main/docs/language/functions.md#compare
type Compare struct {
```